axis_linear: 1  # Left thumb stick vertical
scale_linear: 2.0
scale_linear_turbo: 2.5

axis_angular: 3  # right thumb stick horizontal
scale_angular: 2.0

enable_button: 3  # top trigger button (Y)
enable_turbo_button: -1  # disable trigger button
